main.py

```diff
@@ -1,19 +1,35 @@
-import http.server
 import os
 
-PORT = 8000
+from flask import Blueprint, Flask, send_from_directory
 
-web_dir = os.path.join(os.path.dirname(__file__), 'dist')
-os.chdir(web_dir)
+from app import create_app, notify_celery  # noqa
+from app.render import alerts_from_yaml, get_rendered_pages
 
-Handler = http.server.SimpleHTTPRequestHandler
+dev = Blueprint('dev', __name__)
 
-Handler.extensions_map = {
-    '': 'text/html',
-    '.css': 'text/css'
-}
 
-httpd = http.server.HTTPServer(("", PORT), Handler)
+@dev.route('/<path:key>', methods=['GET'])
+def show_page(key):
+    if os.getenv("FLASK_ENV") != "development":
+        return "not found", 404
 
-print(f"serving at http://localhost:{PORT}")
-httpd.serve_forever()
+    alerts = alerts_from_yaml()
+    rendered_pages = get_rendered_pages(alerts)
+
+    if key not in rendered_pages:
```

Comment:
There's a simpler and more conventional way to do this using Flask config. I made a quick example [here](https://github.com/alphagov/notifications-govuk-alerts/commit/c17a2153f56db46790fbc44b2c8b2d478deab4df#diff-b10564ab7d2c520cdd0243874879fb0a782862c3c902ab535faabe57d5a505e1R38). Can we do that instead?

Reply:
Nice, I force pushed: https://github.com/alphagov/notifications-govuk-alerts/compare/27f6191bdf2a4c1fb3c7f99cfc1e3966d29fc5c4..d45fcfe81b45d7573cb42a5bd1ac2b28313ff6c3